data_architect_misc/data_transformer/transform_functions/eu_latvia_transform_functions.py

```diff
@@ -0,0 +1,26 @@
+"""This is the subclass of Transform function for Latvia (EU division).
+
+We will define transform functions specific to Latvia here.
+
+Author: Maicol Contreras
+Last Modified: December 4, 2020
```

Comment:
You should be updating this comment in the future. Good to know when you worked on it. :)